src/main/scala/org/broadinstitute/dsde/workbench/sam/service/TosService.scala

```diff
@@ -37,10 +38,10 @@ class TosService(val directoryDao: DirectoryDAO, val appsDomain: String, val tos
       directoryDao.loadUser(userId, samRequestContext).map(_.map(_.acceptedTosVersion.contains(tosConfig.version)))
     } else IO.none
 
-  /** If grace period enabled, don't check ToS, return true If ToS disabled, return true Otherwise return true if user has accepted ToS
+  /** If grace period enabled, don't check ToS, return true If ToS disabled, return true Otherwise return true if user has accepted ToS, or is a service account
```

Comment:
Do you think you could just reword this comment while you're in there? I think I know what it's getting at but I'm having trouble parsing it